FreeRTOS/FreeRTOS-Kernel
# Separate the task create function for core affinity

Separate the task create function for core affinity

Description
-----------
Create the following private function to create task:
* prvCreateStaticTask
* prvCreateRestrictedStaticTask
* prvCreateRestrictedTask
* prvCreateTask

uxCoreAffinityMask and prvAddNewTaskToReadyList are still required to be called to setup core affinity and add the task to ready list.

Test Steps
-----------
N/A

Checklist:
----------



Related Issue
-----------
Address #750 in comment 1151, 1625


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
